### PR TITLE
[ADD] Throttling with database load indicator

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

**Review by rconjour — APPROVED** [review]

File: queue_job/models/database_load_indicator.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    arguments = fields.Char(default="()")
+    definition = fields.Text(
+        compute="_compute_definition",
+        help=(
+            "This is the definition as retrieved from the Odoo database. "
+            "You can only update the definition in the database itself."
+        ),
+    )
+
+    def _compute_definition(self):
```
Comment (PieterPaulussen) [inline queue_job/models/database_load_indicator.py:73]: Other methods have a docstring, `_compute_definition` and `_get_active_indicator` don't.

**Review by PieterPaulussen — APPROVED** [review]
Some missing documentation strings. Otherwise a great PR.

File: queue_job/models/database_load_indicator.py (hunk 1 above)
Comment (StefanRijnhart) [reply queue_job/models/database_load_indicator.py:73]: Thanks, docstring added.

**Comment by StefanRijnhart** [comment]
> Some missing documentation strings. Otherwise a great PR.

Thanks for the compliment but all I did was copy over some code :-)

**Pull request merged**